banks/middleware.py

```diff
@@ -17,3 +19,17 @@ def __call__(self, request):
         logger.info(f"Response status: {response.status_code}")
 
         return response
+
+
+class MaintenanceModeMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        if getattr(config, 'MAINTENANCE_MODE', False):
```

Comment:
can we access constance vars like this as in this docs
https://django-constance.readthedocs.io/en/latest/#:~:text=config%0A%0A%23%20...%0A%0Aif-,config.THE_ANSWER,-%3D%3D%2042%3A